tests/jax/test_fused_attn.py

```diff
@@ -147,6 +147,8 @@ def customcall_cross_fused_attn(q, kv, q_token, kv_token, dropout_rng, **kwargs)
     return cross_fused_attn(q, kv, mask, dropout_rng, **kwargs)
 
 
+@pytest.mark.skipif(get_device_compute_capability(0) not in [80, 90],
+                    reason="Fused attention kernel is not supported.")
```

Comment:
Hey, I think there might be some misunderstanding about the supported architecture for fused attention: https://github.com/NVIDIA/TransformerEngine/pull/393#discussion_r1300617951

The max512 backend doesn't support sm86/89 but the arbitrary_seqlen backend does, since cuDNN 8.9.3.

I think you can use something like this to check whether the compute capability condition is met. (This should work for PyTorch at least, Jax might have its own extension with a similar name.)

https://github.com/NVIDIA/TransformerEngine/blob/a150d286a76f6cd4e34b0e16e8b9cc11c4dba34c/transformer_engine/pytorch/attention.py#L1150

Reply:
Using `get_fused_attn_backend` for checking if fused attention supportness has been updated in #393, and it has been merged into the main branch